ingest.py

```diff
@@ -1,53 +1,45 @@
 #!/bin/env python
-import tarfile
 import fastavro
 import sys
 import io
 import boto3
 import os
 import logging
 import requests
-import dramatiq
 import base64
 import time
-import redis
-from dramatiq.brokers.redis import RedisBroker
+from kafka import KafkaConsumer
 from astropy.coordinates import SkyCoord
 from astropy.time import Time
 
 from ztf import Alert, db, app
 
-REDIS_HOST = os.getenv('REDIS_HOST', '127.0.0.1')
-r = redis.StrictRedis(host=REDIS_HOST, charset='utf-8', decode_responses=True)
-redis_broker = RedisBroker(url=f'redis://{REDIS_HOST}:6379/0')
-dramatiq.set_broker(redis_broker)
-
-AWS_ACCESS_KEY = os.getenv('AWS_ACCESS_KEY_ID')
```

Comment:
I know boto checks these env variables behind the scenes on it's own, but as someone that doesn't know that I'd be pretty confused as to how to set the credentials. Can we leave these in to be explicit?

Reply:
Yeah, makes sense. Figured I'd play defense on those, but you're right, it's clearer.